content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md

```diff
@@ -54,29 +54,21 @@ cilium install
 ```
 🔮 Auto-detected Kubernetes kind: minikube
```

Comment:
Hi @Kikiodazie.
As the `cilium` command-line tool changes, the output and docs also must be modified. Does it make sense to remove the sample `cilium install` output? You could refer to the Cilium documentation for a sample of the expected output.

What do you think about separating the curl and extract commands from the `cilium install` command, and provide an explanation of what happens during the installation.

```shell
cilium install
```

Reply:
That makes sense @kbhawkey will make the edit with your feedback.

Thank you @kbhawkey.